src/platform/packages/shared/deeplinks/search/constants.ts

```diff
@@ -11,6 +11,8 @@ export const ENTERPRISE_SEARCH_APP_ID = 'enterpriseSearch';
 export const ENTERPRISE_SEARCH_CONTENT_APP_ID = 'enterpriseSearchContent';
 export const ENTERPRISE_SEARCH_APPLICATIONS_APP_ID = 'enterpriseSearchApplications';
 export const ENTERPRISE_SEARCH_ANALYTICS_APP_ID = 'enterpriseSearchAnalytics';
+export const SEARCH_INDICES_START_APP_ID = 'elasticsearchStart';
```

Comment:
These are already defined on line 23 & 24 as `SEARCH_INDICES_START` & `SEARCH_INDICES` can we just use those instead of introducing duplicate constants?

Additionally if we were adding these as new constants you would have also needed to add these to the types in the `deep_links.ts` file along with adding them to this file.